msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Meeting app extensibility
+author: laujan
+description: Understand the meeting app extensibility 
+ms.topic: conceptual
+---
+
+# Meeting app extensibility
+
+Teams’ meeting app extensibility is based on the following concepts:
+
+✔ Meeting lifecycle has stages such as before, during, and after meeting time frame.  
+✔ Participant roles in a meeting such as meeting organizer, presenter, or attendee.  
```

Comment:
Reword
There are three distinct roles in a meeting; organizer, presenter, and attendee. [Learn more](https://support.microsoft.com/en-us/office/roles-in-a-teams-meeting-c16fa7d0-1666-4dde-8686-0a0bfe16e019) about these roles